features-json/picture-in-picture.json

```
@@ -175,7 +183,7 @@
       "66":"n",
       "67":"n",
       "68":"n",
-      "69":"n",
+      "69":"d #2",
```

```suggestion
      "69":"n d #2",
```
Need to include the "n" value